AcceptanceTests/AcceptanceTests.csproj

```diff
@@ -7,6 +7,10 @@
   </PropertyGroup>
 
   <ItemGroup>
+    <PackageReference Include="coverlet.collector" Version="1.2.0">
+      <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
+      <PrivateAssets>all</PrivateAssets>
+    </PackageReference>
     <PackageReference Include="Microsoft.Extensions.Configuration.EnvironmentVariables" Version="3.1.2" />
     <PackageReference Include="nunit" Version="3.12.0" />
     <PackageReference Include="NUnit3TestAdapter" Version="3.16.1" />
```

Comment:
This is currently not run a part of CI-build pipeline because I didn't have time for it yet.

If you wish to generate those coverage reports you can use `dotnet test --collect:"XPlat Code Coverage"`, that generates Cobertura XML which is supported by codecov.